packages/client/src/App.tsx

```diff
@@ -87,7 +93,11 @@ const Layout = () => {
       <Route path={pathData['state'].path}>
         <Blurb/>
         <TallStyleContainer>
-          <AddressForm/>
+            <FieldsContainer.Provider>
```

Comment:
This is a mistake -- you should have `<StateForm/>` here (see master branch).  Are you rebasing from the right branch?